lib/validator/validator.go

```diff
@@ -30,14 +32,35 @@ import (
 var validate *validator.Validate
 
 var (
-	nameRegex          = regexp.MustCompile("^[a-zA-Z0-9_.-]+$")
+	nameRegex          = regexp.MustCompile("^[a-zA-Z0-9_.-]{1,128}$")
 	interfaceRegex     = regexp.MustCompile("^[a-zA-Z0-9_-]{1,15}$")
-	labelRegex         = regexp.MustCompile("^[a-zA-Z_./-][a-zA-Z0-9_./-]*$")
+	labelRegex         = regexp.MustCompile("^[a-zA-Z_./-][a-zA-Z0-9_./-]{0,127}$")
```

Comment:
why is this `{0,127}` starting from `0` compared to `nameRegex`? does that mean we allow empty labels?